pages.ja/common/ab.md

```diff
@@ -0,0 +1,24 @@
+# ab
+
+> Apache HTTP server のベンチマーク用ツールです。
+> 詳細情報はこちら: <https://httpd.apache.org/docs/current/programs/ab.html>.
+
+- 指定された URL に 100 個の GET リクエストを送信する:
+
+`ab -n {{100}} {{url}}`
+
+- 指定された URL に 100 個の GET リクエストを、10 個ずつ同時に送信する:
+
+`ab -n {{100}} -c {{10}} {{url}}`
+
+- ファイルからの JSON ペイロードを使って、指定された URL に 100 個の POST リクエストを送信する:
+
+`ab -n {{100}} -T {{application/json}} -p {{path/to/file.json}} {{url}}`
```

Comment:
```suggestion
`ab -n {{100}} -T {{application/json}} -p {{ファイルパス.json}} {{url}}`
```
Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md

Reply:
@kbdharun 
Thank you for comment. I have translated the placeholder.